contents/handbook/engineering/feature-ownership.md

```diff
@@ -15,35 +15,38 @@ You can also view the list [directly in GitHub](https://github.com/PostHog/posth
-| Cohorts | [@EDsCODE][@EDsCODE]  |  <span class="lemon-tag gh-tag">cohorts</span>  |
-| Correlation Analysis | [@neilkakkar][@neilkakkar]  |  <span class="lemon-tag gh-tag">correlation</span> |
-| Dashboards | [@Twixes][@Twixes]  |  <span class="lemon-tag gh-tag">dashboards</span> |
-| Events | [@alexkim205][@alexkim205]  |  <span class="lemon-tag gh-tag">events</span>  |
-| Experimentation | [@neilkakkar][@neilkakkar] |  <span class="lemon-tag gh-tag">experimentation</span> |
-| Feature Flags | [@neilkakkar][@neilkakkar]  |  <span class="lemon-tag gh-tag">feature-flags</span> |
-| Funnels | [@neilkakkar][@neilkakkar]  |  <span class="lemon-tag gh-tag">funnels</span>  |
-| Group Analytics | [@macobo][@macobo]  |  <span class="lemon-tag gh-tag">group-analytics</span> |
-| Onboarding | [@liyiy][@liyiy]  | <span class="lemon-tag gh-tag">onboarding</span>  |
-| Lifecycle | [@EDsCODE][@EDsCODE]  | <span class="lemon-tag gh-tag">lifecycle</span>  |
-| Paths | [@neilkakkar][@neilkakkar]  |  <span class="lemon-tag gh-tag">paths</span> |
-| Permissions | [@Twixes][@Twixes]  | <span class="lemon-tag gh-tag">permissions</span>  |
-| Persons | [@macobo][@macobo]  | <span class="lemon-tag gh-tag">persons</span>  |
-| Project Home Page | [@rcmarron][@rcmarron]  | <span class="lemon-tag gh-tag">home</span> |
-| Property Filters | [@mariusandra][@mariusandra]  | <span class="lemon-tag gh-tag">filters</span>  |
-| Recordings | [@rcmarron][@rcmarron]  |  <span class="lemon-tag gh-tag">recordings</span> |
-| Retention | [@EDsCODE][@EDsCODE]  |  <span class="lemon-tag gh-tag">retention</span> |
-| Saved Insights | [@Twixes][@Twixes]  |  <span class="lemon-tag gh-tag">saved-insights</span> |
-| Session Analytics | [@rcmarron][@rcmarron]  |  <span class="lemon-tag gh-tag">sessions</span> |
-| Settings (personal & project) | [@liyiy][@liyiy]  |  <span class="lemon-tag gh-tag">settings</span> |
-| Stickiness | [@EDsCODE][@EDsCODE]  | <span class="lemon-tag gh-tag">stickiness</span>  |
-| Taxonomy | [@alexkim205][@alexkim205]  | <span class="lemon-tag gh-tag">taxonomy</span>  |
-| Toolbar | [@pauldambra][@pauldambra]  | <span class="lemon-tag gh-tag">toolbar</span>  |
-| Trends | [@EDsCODE][@EDsCODE]  | <span class="lemon-tag gh-tag">trends</span>  |
+| Actions | [@pauldambra][@pauldambra]  | <span class="lemon-tag gh-tag">feature/actions</span> |
+| Actors Modal | [@EDsCODE][@EDsCODE]  | <span class="lemon-tag gh-tag">feature/actors-modal</span>  |
+| Annotations | [@pauldambra][@pauldambra]  | <span class="lemon-tag gh-tag">feature/annotations</span> |
+| API Structure | [@Twixes][@Twixes]  | <span class="lemon-tag gh-tag">feature/api-structure</span>  |
+| Application Performance Monitoring (APM) | [@pauldambra][@pauldambra]  | <span class="lemon-tag gh-tag">feature/apm</span>  |
+| Async migrations | [@yakkomajuri][@yakkomajuri]  | <span class="lemon-tag gh-tag">feature/async-migrations</span> |
```

Comment:
CC @yakkomajuri

Reply:
Well, so we discussed this with the Platform team and we don't actually want to have feature owners (for various reasons). If this holds back the rest of the team because they want to know who owns what we can consider some ideas.

However, we've split the team into Ingestion and Infrastructure - both notably missing from here.

Ingestion is me and @tiina303. Infra is @guidoiaquinti and @hazzadous.

Async migrations is the one "feature" out of place, because Ingestion owns it (for now).

Maybe I can add team platform's features here but linking to the respective team pages? So they'll link to an updated page with the current team members?

Reply:
Makes sense @yakkomajuri, I added context to this page to clarify the way Team Platform works. Feel free to make any edits necessary.